client/blocks/inline-help/inline-help-search-card.jsx

```diff
@@ -6,6 +6,7 @@ import { useDispatch } from 'react-redux';
 import SearchCard from 'calypso/components/search-card';
 import { useHelpSearchQuery } from 'calypso/data/help/use-help-search-query';
 import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import './inline-help-search-card.scss';
```

Comment:
The `inline-help__search` classname under which all CSS inside this stylesheet is nested is not being used directly in this component, so at first glance it feels odd to import this stylesheet here.

Reply:
Yes, it's used as:
```jsx
<div className="inline-help__search">
  <HelpSearchCard />
  <HelpSearchResults />
</div>
```
It doesn't have its own component where the styles could live, but always travels together with the `HelpSearchCard` component. So including the styles there is a dirty trick, but it works. Can be cleanup as one of follow-ups.